Capstone/Autocorrect/AutocorrectFunctions.m

```diff
@@ -0,0 +1,114 @@
+//
+//  AutocorrectFunctions.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/14/22.
+//
+#import "AutocorrectFunctions.h"
+#include "math.h"
+#import "College.h"
+
+@implementation AutocorrectFunctions
+
++ (NSString *)findCorrectWord:(NSString *)searchWord forCollegesInArray:(NSArray *)colleges {
+    AutocorrectFunctions *myAutocorrectFunctions = [[AutocorrectFunctions alloc] init];
+    double minDifferenceBetweenWords = 1000;
+    NSString *correctWord;
+    for (College *college in colleges) {
+        double hammingDistanceFullCollegeName = [myAutocorrectFunctions hammingDistanceCalculator:searchWord secondWordToCompareOutOfTwoWords:college.name];
```

Comment:
I don't think you need to create an AutocorrectFunction object if your helpers (`hammingDistanceCalculator:secondWordToCompareOutOfToWords:`) are static also
```suggestion
        double hammingDistanceFullCollegeName = [self hammingDistanceCalculator:searchWord secondWordToCompareOutOfTwoWords:college.name];
```